http/pprof/pprof.go

```diff
@@ -0,0 +1,47 @@
+package pprof
+
+import (
+	"fmt"
+	"github.com/pyroscope-io/godeltaprof"
+	"io"
+	"net/http"
+	"runtime"
+	"strconv"
+)
+
+var deltaHeapProfiler = godeltaprof.NewHeapProfiler()
+var deltaBlockProfiler = godeltaprof.NewBlockProfiler()
+var deltaMutexProfiler = godeltaprof.NewMutexProfiler()
+
+type deltaProfiler interface {
+	Profile(w io.Writer) error
+}
+
+func init() {
+	http.HandleFunc("/debug/pprof/delta_heap", Heap)
+	http.HandleFunc("/debug/pprof/delta_block", Block)
+	http.HandleFunc("/debug/pprof/delta_mutex", Mutex)
+}
+
+func Heap(w http.ResponseWriter, r *http.Request) {
+	WriteDeltaProfile(&deltaHeapProfiler, "heap", w, r)
+}
```

Comment:
Should we support `gc=1` query parameter for forcing GC?

Reply:
yes, let's support gc=1, just in case someone want to use it later
done

Reply:
My bad: `WriteDeltaProfile` already does this check and calls GC, we should keep only one. I'd prefer to have this logic in the `heap` endpoint, just for clarity

Reply:
removed